apps/site/lib/site_web/templates/schedule/_line.html.eex

```diff
@@ -15,11 +15,29 @@
         <%= if Enum.empty?(@branches) do %>
           <%= render "_empty.html", date: @date, direction: Routes.Route.direction_name(@route, @direction_id), origin: nil, destination: nil, conn: @conn, error: assigns[:schedule_error] %>
         <% else %>
-
-          <h2><%= if @route.type == 2, do: "Schedule", else: "Schedules" %> &amp; Maps</h2>
+          <% title = case @route.type do
+              2 -> "Schedule & Maps"
+              4 -> nil
+              _ -> "Schedules & Maps"
+            end %>
+          <%= if title do %>
+            <h2><%= title %></h2>
+          <% end %>
 
           <div id="react-schedule-direction-root"></div>
 
+          <div id="react-schedule-finder-root">
```

Comment:
suggestion: Combine this logic into the #react-schedule-finder-root element at line 69. And we don't want to end up with two `<div id="react-schedule-finder-root">` elements.

Reply:
Ok, I couldn't figure out how to combine them... because the placement of each chunk of code has layout implications (but if there's a way to put component-alize the chunk, I'm all-ears!)  But I DID fix the duplicate react-schedule-finder-root elements (which fixed the fact that the schedule-finder was unusuable on some pages 😅)

Reply:
That works, yay!